binder/src/main/scala/PanamaCIRCT.scala

```diff
@@ -470,6 +508,62 @@ class PanamaCIRCT {
+  }
+  def omEvaluatorGetModule(evaluator: OMEvaluator) = MlirModule(CAPI.omEvaluatorGetModule(arena, evaluator.get))
+  def omEvaluatorObjectIsNull(obj: OMEvaluatorValue): Boolean = CAPI.omEvaluatorObjectIsNull(obj.get)
+  def omEvaluatorObjectGetType(obj: OMEvaluatorValue) = MlirType(CAPI.omEvaluatorObjectGetType(arena, obj.get))
+  def omEvaluatorObjectGetField(obj: OMEvaluatorValue, name: String) = OMEvaluatorValue(CAPI.omEvaluatorObjectGetField(arena, obj.get, mlirStringAttrGet(name).get))
+  def omEvaluatorObjectGetHash(obj: OMEvaluatorValue): Int = CAPI.omEvaluatorObjectGetHash(obj.get)
+  def omEvaluatorObjectIsEq(obj: OMEvaluatorValue, other: OMEvaluatorValue): Boolean = CAPI.omEvaluatorObjectIsEq(obj.get, other.get)
+  def omEvaluatorObjectGetFieldNames(obj: OMEvaluatorValue) = MlirAttribute(CAPI.omEvaluatorObjectGetFieldNames(arena, obj.get))
+  def omEvaluatorValueGetLoc(evaluatorValue: OMEvaluatorValue) = MlirLocation(CAPI.omEvaluatorValueGetLoc(arena, evaluatorValue.get))
+  def omEvaluatorValueIsNull(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsNull(evaluatorValue.get)
+  def omEvaluatorValueIsAObject(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsAObject(evaluatorValue.get)
+  def omEvaluatorValueIsAPrimitive(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsAPrimitive(evaluatorValue.get)
+  def omEvaluatorValueGetPrimitive(evaluatorValue: OMEvaluatorValue) = MlirAttribute(CAPI.omEvaluatorValueGetPrimitive(arena, evaluatorValue.get))
+  def omEvaluatorValueFromPrimitive(primitive: MlirAttribute) = OMEvaluatorValue(CAPI.omEvaluatorValueFromPrimitive(arena, primitive.get))
+  def omEvaluatorValueIsAList(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsAList(evaluatorValue.get)
+  def omEvaluatorListGetNumElements(evaluatorValue: OMEvaluatorValue): Int = CAPI.omEvaluatorListGetNumElements(evaluatorValue.get).toInt
+  def omEvaluatorListGetElement(evaluatorValue: OMEvaluatorValue, pos: Int) = OMEvaluatorValue(CAPI.omEvaluatorListGetElement(arena, evaluatorValue.get, pos))
+  def omEvaluatorValueIsATuple(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsATuple(evaluatorValue.get)
+  def omEvaluatorTupleGetNumElements(evaluatorValue: OMEvaluatorValue): Int = CAPI.omEvaluatorTupleGetNumElements(evaluatorValue.get).toInt
+  def omEvaluatorTupleGetElement(evaluatorValue: OMEvaluatorValue, pos: Int) = OMEvaluatorValue(CAPI.omEvaluatorTupleGetElement(arena, evaluatorValue.get, pos))
+  def omEvaluatorMapGetElement(evaluatorValue: OMEvaluatorValue, attr: MlirAttribute) = OMEvaluatorValue(CAPI.omEvaluatorMapGetElement(arena, evaluatorValue.get, attr.get))
+  def omEvaluatorMapGetKeys(obj: OMEvaluatorValue) = MlirAttribute(CAPI.omEvaluatorMapGetKeys(arena, obj.get))
+  def omEvaluatorValueIsAMap(evaluatorValue: OMEvaluatorValue): Boolean = CAPI.omEvaluatorValueIsAMap(evaluatorValue.get)
```

Comment:
Map still remains?

Reply:
They are API wrappers. Since we don't use them anymore, it's fine to remove them form Chisel as well. I'll do it later.

Reply:
Removed `firrtlTypeGetMap`.